app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
```

Comment:
Consider changing this to hourOpen and adding an hourClose property.  This will allow you to use the hourClose property to determine the stopping point of your loops.  If possible, avoid using hard-coded numbers to control loops (except priming a loop e.g. var i = 0).